dotcom-rendering/index.d.ts

```diff
@@ -696,7 +696,7 @@ type LineEffectType = 'labs' | 'dotted' | 'straight';
 
 type LeftColSize = 'compact' | 'wide';
 
-type CardPercentageType = '25%' | '34%' | '50%' | '66%' | '75%' | '100%';
+type CardPercentageType = '25%' | '33.333%' | '50%' | '66%' | '75%' | '100%';
```

Comment:
This change was necessary for the layout to look correct, from examining other areas where this % was used, I don't think we'll see a big impact on the site from it

Reply:
I haven't tested this - are there browsers which don't support flex-basis being set to decimals? From what I remember I saw frontend use decimals so I feel it should be okay?